packages/frontend/src/pages/ImportModels.svelte

```diff
@@ -0,0 +1,147 @@
+  const target = event.currentTarget as HTMLInputElement;
+  modelsToImport[index].name = target.value;
+  modelsToImport = modelsToImport;
+}
+
+function deleteModelToImport(index: number) {
+  modelsToImport = modelsToImport.filter((_, i) => i !== index);
+}
+
+async function importModels() {
+  importError = '';
+  inProgress = true;
+
+  try {
+    await studioClient.importModels(modelsToImport);
+  } catch (e) {
+    importError = `Error while importing models: ${String(e)}\n`;
+  }
+
+  inProgress = false;
+  if (importError === '') {
+    router.goto(`/models`);
+  }
+}
+</script>
+
+<NavPage lastPage="{{ name: 'Models', path: '/models' }}" title="Import Models" searchEnabled="{false}">
+  <svelte:fragment slot="content">
+    <div class="p-5 min-w-full h-fit">
+      <div class="bg-charcoal-800 rounded-lg px-6 py-4 space-y-2">
+        <Button on:click="{addModelsToImport}" icon="{faPlusCircle}" type="link" aria-label="Add models"
+          >Add .GGUF Models</Button>
+        <div aria-label="importError">
+          {#if addModelError !== ''}
+            <ErrorMessage class="py-2 text-sm" error="{addModelError}" />
+          {/if}
+        </div>
+        <!-- Display the list of existing containersToImport -->
+        {#if modelsToImport.length > 0}
+          <div class="flex flex-row justify-center w-full py-1 text-sm font-medium text-gray-400">
+            <div class="flex flex-col grow pl-2">Path</div>
+            <div class="flex flex-col w-2/4 mr-2.5">Name</div>
+          </div>
+        {/if}
+        {#each modelsToImport as modelToImport, index}
+          <div class="flex flex-row w-full py-1">
+            <div class="flex flex-col grow">
+              <input
+                class="flex flex-col grow p-2 outline-none text-sm bg-transparent rounded-sm text-gray-700 placeholder-gray-700 border-charcoal-100 border-b mr-2"
+                bind:value="{modelToImport.path}"
+                aria-label="model path"
+                readonly="{true}" />
+            </div>
+            <div class="flex flex-col grow">
+              <input
+                bind:value="{modelToImport.name}"
+                on:input="{event => onModelNameInput(event, index)}"
+                aria-label="model importing name"
+                placeholder="Model Name displayed"
+                class="flex flex-col grow ml-2 p-1 outline-none text-sm bg-transparent rounded-sm text-gray-700 placeholder-gray-700 border-charcoal-100 border-b hover:border-purple-400 focus:border-purple-400 focus:border focus:rounded-lg focus:bg-charcoal-900" />
+            </div>
+            <Button type="link" on:click="{() => deleteModelToImport(index)}" icon="{faMinusCircle}" />
+          </div>
+        {/each}
+
+        <div aria-label="importError">
+          {#if importError !== ''}
+            <ErrorMessage class="py-2 text-sm" error="{importError}" />
+          {/if}
+        </div>
+        <Button
```

Comment:
```suggestion
        <div class="pt-5 border-zinc-600 border-t-2"/>
        <Button
```

Adding a divider to match the design container import

Reply:
So the container import page has to be updated. I forgot and i'll create a PR for this on desktop. When working on the load images page i was copying the container import template and Tim commented to remove the divider as it is not used anywhere else https://github.com/containers/podman-desktop/pull/6540#discussion_r1539272375

Reply:
Okey np

Reply:
Thanks for making me remind about it 😆 -> https://github.com/containers/podman-desktop/pull/6655